src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/PersonAttendingSummaryReportITest.java

```diff
@@ -0,0 +1,265 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@Sql({
+    "/db/mod/truncate.sql",
+    "/db/mod/reports/PersonAttendingSummaryReportITest_typical.sql"
+})
+@SuppressWarnings({
+    "PMD.LawOfDemeter",
+    "PMD.JUnitTestsShouldIncludeAssert"//False positive
+})
+class PersonAttendingSummaryReportITest extends AbstractStandardReportControllerITest {
+    @Autowired
+    public PersonAttendingSummaryReportITest(TestRestTemplate template) {
+        super(template, PersonAttendingSummaryReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getCourtJwt("415");
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .reportType("PersonAttendingSummaryReport")
```

Comment:
Not needed as this will be added by the addReportType(...) method

Reply:
Fixed